docs/resources/alerting_rule.md

```diff
@@ -1,5 +1,4 @@
 ---
-# generated by https://github.com/hashicorp/terraform-plugin-docs
```

Comment:
I removed this line because it kept tripping me up thinking that these files were generated _every time_ as opposed to the reality that they were initially generated as convenient placeholders but from that point onward they are manually edited by us.

Reply:
are these files not updated by running the tfplugindocs command the DEVELOPMENT.md file describes?